app/controllers/games_controller.rb

```diff
@@ -43,6 +48,8 @@ def update
   end
 
   def destroy
+    @game.game_players.destroy_all
+    @game.rounds.destroy_all
```

Comment:
Do you still need these, since you added `dependent: :delete_all`?

Reply:
I've removed `@game.rounds.destroy_all`  but we need to keep the `@game.game_players.destroy_all` because there is a foireign key in rounds table from game_players and it violates the foreign key constraint